code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -558,6 +556,8 @@
 		return
 	if(mode == BOT_IDLE)
 		return
+	if(HAS_TRAIT(src, TRAIT_IMMOBILIZED))
+		return
 
 	var/speed = (wires.is_cut(WIRE_MOTOR1) ? 0 : 1) + (wires.is_cut(WIRE_MOTOR2) ? 0 : 2)
```

Comment:
It looks like this is still doing its own work, despite the varspeed being set? Do you still need this?

Reply:
This disables movement if both motor wires have been cut or if the mulebot has been turned off.